lib/types.ts

```diff
@@ -10,13 +10,18 @@ export interface ICardData {
   level: number
   name: string
   type: string
+  cost?: string | undefined
   rarity?: string | undefined
   color?: string | undefined
   attack?: string | undefined
   defense?: string | undefined
-  effect?: string | undefined
-  specialEffect?: string | undefined
   ability_1?: string | undefined
   ability_2?: string | undefined
+  effect1?: string | undefined
+  effect2?: string | undefined
+  effect3?: string | undefined
+  // not currently used for game
+  effect?: string | undefined
```

Comment:
having `effect` and `effect1` ... `effect3` has suspicious code smell.

Reply:
Ya this is trash - again, this is me bending the rules to support usage of the spreadsheet. The community (@0S) originally contributed the "effect" field, so I didn't want to remove it.

Reply:
not to go too philosophical here, but this parable comes to mind: https://www.learnreligions.com/the-buddhas-raft-parable-450054

i.e. the raft that got us here doesn't need to be the one we take with us.

thank you for your service `effect` field.

and @0s, if you're reading this, we look forward to your company on this `0xAdventure`